mitosheet/mitosheet/public/v3/sheet_functions/number_functions.py

```diff
@@ -488,21 +488,39 @@ def ROUND(arg: NumberRestrictedInputType, decimals: Optional[IntRestrictedInputT
     }
     """
 
+    def excel_round(x: Optional[Union[int, float]], decimals: int) -> Optional[Union[int, float]]:
+        if pd.isna(x): 
+            return None 
+
+        factor = 10 ** decimals
+        x *= factor
```

Comment:
This needs some serious documentation. What is going on here? Is there a better name we can pick for factor?